dask_match/datasets.py

```diff
@@ -12,7 +12,16 @@
 
 
 class Timeseries(BlockwiseIO):
-    _parameters = ["start", "end", "dtypes", "freq", "partition_freq", "seed", "kwargs"]
+    _parameters = [
+        "start",
+        "end",
+        "dtypes",
+        "freq",
+        "partition_freq",
+        "seed",
+        "kwargs",
+        "_take_partitions",
```

Comment:
Thoughts on other names?  Maybe just `_partitions`?

Reply:
I used `_partitions` at first be went with something more unique to avoid confusion with the collection-level `_partitions`. Though, I'll admit I like the simplicity of `_partitions` better. So, very open to changing it :)